api/src/test_utils.rs

```diff
@@ -55,3 +28,20 @@ pub fn new_test_context() -> Context {
 
     Context::new(ChainId::test(), db)
 }
+
+pub async fn send_request(context: Context, method: &str, path: &str, status_code: u16) -> Value {
+    let filter = index::routes(context);
```

Comment:
Why name this filter and not, say, `handler` or `router`?

Reply:
because in warp it is a Filter instance. 
Let me change it to routes.